keen/explorer
# Use webpack instead of gulp

### What does this PR do?

This PR replaces some of our gulp build steps with webpack and adds npm scripts for our common gulp tasks. We're switching to webpack to standardize and simplify our build process for our JS projects.

This PR also updates pickadate (because the version we used was incompatible with webpack), and fixes some accidental global variables in our codebase.

### How to test:
* Run `npm install`. There are quite a few new modules.
* Run
  * `npm run dev`
  * `npm run build`
  * `npm run minify`
  * `npm run production`
and make sure they work correctly.
